include/class.organization.php

```diff
@@ -436,7 +436,7 @@ function delete() {
 
         // Remove users from this organization
         User::objects()
-            ->filter(array('org' => $this))
```

Comment:
I see. Yeah, the join would fail. Perhaps the ORM should use the local side of the relationship if possible (automatically what your change proposes).